user_guide_src/source/outgoing/api_responses.rst

```diff
@@ -49,6 +49,8 @@ format both XML and JSON responses:
 
 .. literalinclude:: api_responses/003.php
 
+.. note:: Since ``v4.7.0``, you can change the default JSON formatting depth by editing **app/Config/Format.php**. The ``$jsonDepthOptions`` value defines the maximum depth, with a default of ``512``.
```

Comment:
```suggestion
.. note:: Since ``v4.7.0``, you can change the default JSON encoding depth by editing **app/Config/Format.php** file. The ``$jsonEncodeDepth`` value defines the maximum depth, with a default of ``512``.
```